yaml/lang/fr.yml

```diff
@@ -0,0 +1,430 @@
+ACTOR:
+  TypeCharacter: Personnage
+  TypeMinion: Sbire
+  TypeRival: Rival
+  TypeNemesis: Nemesis
+  TypeVehicle : Vehicule
+
+ITEM:
+  # Character Creation Items
+  TypeAbility: Special Ability
+  TypeArchetype: Archetype
+  TypeCareer: Career
+  TypeInjury: Critical Injury/Hit
```

Comment:
These don't seem to be translated.

Reply:
Yes, in my tanslate, i have and the yml it's not : i correct that